coins.json

```diff
@@ -1,1408 +1,1493 @@
-            "sampleTx": "bafy2bzacecbm3ofxjjzcl2rg32ninphza34mm3ijr55zjsamwfqmz4ib63mqe",
-            "sampleAccount": "t1nbb73vhk5dtmnsgeaetbo76daepqjtrfoccn74i"
-        },
-        "info": {
-            "url": "https://filecoin.io/",
-            "client": "https://github.com/filecoin-project/lotus",
-            "clientPublic": "",
-            "clientDocs": "https://docs.lotu.sh"
-        }
-    },
-    {
-        "id": "elrond",
-        "name": "Elrond",
-        "symbol": "ERD",
-        "decimals": 18,
-        "blockchain": "ElrondNetwork",
-        "derivationPath": "m/44'/508'/0'/0'/0'",
-        "curve": "ed25519",
-        "publicKeyType": "ed25519",
-        "hrp": "erd",
-        "explorer": {
-            "url": "https://explorer.elrond.com",
-            "txPath": "/transactions/",
-            "accountPath": "/address/"
-        },
-        "info": {
-            "url": "https://elrond.com/",
-            "client": "https://github.com/ElrondNetwork/elrond-go",
-            "clientPublic": "https://api.elrond.com",
-            "clientDocs": "https://docs.elrond.com"
-        }
-    },
-    {
-        "id": "band",
-        "name": "BandChain",
-        "symbol": "BAND",
-        "decimals": 6,
-        "blockchain": "Cosmos",
-        "derivationPath": "m/44'/494'/0'/0/0",
-        "curve": "secp256k1",
-        "publicKeyType": "secp256k1",
-        "hrp": "band",
-        "explorer": {
-            "url": "https://scan-wenchang-testnet2.bandchain.org/",
-            "txPath": "/tx/",
-            "accountPath": "/account/",
-            "sampleTx": "473264551D3063A9EC64EC251C61BE92DDDFCF6CC46D026D1E574D83D5447173",
-            "sampleAccount": "band12nmsm9khdsv0tywge43q3zwj8kkj3hvup9rltp"
-        },
-        "info": {
-            "url": "https://bandprotocol.com/",
-            "client": "https://github.com/bandprotocol/bandchain",
-            "clientPublic": "https://api-wt2-lb.bandchain.org",
-            "clientDocs": "https://docs.bandchain.org/"
-        }
-    }
-]
+[
+  {
+    "id": "bitcoin",
+    "name": "Bitcoin",
+    "symbol": "BTC",
+    "decimals": 8,
+    "blockchain": "Bitcoin",
+    "derivationPath": "m/84'/0'/0'/0/0",
+    "curve": "secp256k1",
+    "publicKeyType": "secp256k1",
+    "p2pkhPrefix": 0,
+    "p2shPrefix": 5,
+    "hrp": "bc",
+    "publicKeyHasher": "sha256ripemd",
+    "base58Hasher": "sha256d",
+    "xpub": "zpub",
+    "xprv": "zprv",
+    "explorer": {
+      "url": "https://blockchair.com",
+      "txPath": "/bitcoin/transaction/",
+      "accountPath": "/bitcoin/address/",
+      "sampleTx": "0607f62530b68cfcc91c57a1702841dd399a899d0eecda8e31ecca3f52f01df2",
+      "sampleAccount": "17A16QmavnUfCW11DAApiJxp7ARnxN5pGX"
+    },
+    "info": {
+      "url": "https://bitcoin.org",
+      "client": "https://github.com/trezor/blockbook",
+      "clientPublic": "",
+      "clientDocs": "https://github.com/trezor/blockbook/blob/master/docs/api.md"
+    },
+    "numericId": 0
```

Comment:
Could we have numericId higher up, right next to symbol?

Reply:
this is inserted by a script, I can move that later